<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vckake</class>
 <widget class="QMainWindow" name="vckake">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vckake</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>421</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>84</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>新規</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>84</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>削除</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>947</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
